src/components/forms/comment-form/tests/comment-form.test.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import * as redux from 'react-redux';
+import { render, fireEvent, act } from '@testing-library/react';
+import * as formik from 'formik';
+
+import CommentForm from '../index';
+import { config } from '../../../../configs';
+
+configure({ adapter: new Adapter() });
+const { GO_BACK_TITLE, SAVE_TITLE } = config.buttonTitles;
+const { productInfo } = config.labels.comment;
+
+const mockComment = {
+  text: 'Допоможіть! Дитина помалювала фламастером, нічим не можу вивести.',
+  show: true,
+  product: {
+    _id: 'UID'
+  }
+};
+const mockId = '6047321793650236ddbfb841';
+const mockIsEdit = true;
```

Comment:
please create a file comments.variables.js and get these values from it

Reply:
Done